CHANGES.rst

```diff
@@ -1,6 +1,206 @@
 Traits CHANGELOG
 ================
 
+Release 6.2.0
+-------------
+
+Released: 2021-01-XXX
+
+The Traits library is a foundational component of the Enthought Tool Suite. It
+provides observable, typed attributes for Python classes, making those classes
+suitable for event-driven dataflow programming and for immediate use as models
+for graphical user interfaces, like those provided by the TraitsUI library.
+
+Traits 6.2 is the latest feature release in the Traits 6 series, with several
+improvements and fixes over Traits 6.1.
+
+Highlights of this release
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The Traits examples are now distributed as part of the Traits egg, and
+  are contributed to the ``etsdemo`` application. (The latter is currently
+  distributed as part of TraitsUI.)
```

Comment:
The application is not distributed as part of TraitsUI. It is a separate distribution installable with `pip install etsdemo`.

Reply:
Thank you! Fixed.